src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,6 @@
+@mixin block($side) {
+  background: #eee;
+  width: $side;
+  height: $side;
+  border: 1px solid black;
```

Comment:
The border color should be a variable, e.g., $cell-border-color, to make it easier to change and maintain.